src/ray/stats/metric_defs.cc

```diff
@@ -67,6 +67,21 @@ DEFINE_stats(actors,
              (),
              ray::stats::GAUGE);
 
+/// Job related stats.
+DEFINE_stats(running_jobs,
+             "Number of jobs currently running.",
+             /*tags=*/(),
+             /*buckets=*/(),
+             ray::stats::GAUGE);
+
+DEFINE_stats(finished_jobs,
+             "New finished jobs number.",
```

Comment:
"Number of jobs finished."

Reply:
Thanks! That's definitely better, updated.